### PR TITLE
[RM Ch-A] Hardware Coding Policy Initial Framework (3 of 3)

### LINKED ISSUE #746: [RM Ch-various] Form Principles Related to HW Abstraction and Exposure Thereof
This Issue is in support of Issue #731.

Proposes principles to support cloud native workload development by stipulating h/w abstraction, and it's application to higher-level features such as PCI-PT, SRIOV and EPA.

### PR DESCRIPTION
**Pull request by markshostak**
Fixes #746.

### CODE REVIEW

**Comment by markshostak** [comment]
Approved with changes during RM Core live review on 4/12/2019. Pending cool-off/comment period.

File: doc/ref_model/chapters/appendix-a.md [hunk 1]
```diff
@@ -5,8 +5,9 @@
 ## Table of Contents
 * [A.1 Goals](#A.1)
 * [A.2 Intro and Terminology](#A.2)
-* [A.3 VNF Maturity Levels](#A.3)
+* [A.3 VNF Evolution Phases](#A.3)
```
Comment (nickchase) [inline doc/ref_model/chapters/appendix-a.md]: I prefer "Phases of Evolution" but not enough to hold things up.

**Comment by rabiabdel** [comment]
Modified language is expected in future PRs and clear linking between compromises and evolution phases.

File: doc/ref_model/chapters/appendix-a.md (hunk 1 above)
Comment (markshostak) [reply doc/ref_model/chapters/appendix-a.md]: @nickchase Hi Nick, I was just making a small change to the language, but the actual implementation is being done under a different PR, so I have interjected you comment in said PR, at the following link: https://github.com/cntt-n/CNTT/pull/754#issuecomment-562574042.

If this action addresses your concern with regard to this PR, please mark this sub-thread as Resolved. Thanks, -Mark

**Pull request merged**